backend/btrixcloud/orgs.py

```diff
@@ -407,7 +407,13 @@ async def rename_org(
         org: Organization = Depends(org_owner_dep),
     ):
         org.name = rename.name
-        await ops.update(org)
+        try:
+            await ops.update(org)
+        except DuplicateKeyError:
+            # pylint: disable=raise-missing-from
+            raise HTTPException(
+                status_code=400, detail=f'Name "{rename.name}"" already taken'
```

Comment:
```suggestion
                status_code=400, detail='duplicate_org_name'
```

Should return error codes so can display localized messages on the client.